src/framing.rs

```diff
@@ -246,9 +225,8 @@ impl MLSCiphertext {
             + 2
             + TAG_BYTES
             + 4;
-        let mut padding_length = (config.padding_block_size as usize)
-            - (padding_offset % (config.padding_block_size as usize));
-        if (config.padding_block_size as usize) == padding_length {
+        let mut padding_length = PADDING_SIZE - (padding_offset % PADDING_SIZE);
+        if PADDING_SIZE == padding_length {
```

Comment:
yeah... this is `let padding_length = (PADDING_SIZE - (padding_offset % PADDING_SIZE)) % PADDING_SIZE;` I guess.
I just complain about this because the `mut` here shouldn't be necessary.